gcpy/benchmark/modules/run_1yr_tt_benchmark.py

```diff
@@ -1197,50 +1252,57 @@ def gchp_vs_gchp_mass_table(mon):
     if config["options"]["outputs"]["cons_table"]:
 
         # ===================================================================
-        # Create mass conservation table for GCC_dev
+        # Create mass conservation table for GCC vs GCC
         # ===================================================================
-        if (
-            config["options"]["comparisons"]["gcc_vs_gcc"]["run"]
-            or config["options"]["comparisons"]["gchp_vs_gcc"]["run"]
-        ):
-            print("\n%%% Creating GCC dev mass conservation table %%%")
+        if config["options"]["comparisons"]["gcc_vs_gcc"]["run"]:
+            print("\n%%% Creating GCC mass conservation tables %%%")
 
             # Filepaths
-            datafiles = get_filepaths(
+            ref_datafiles = get_filepaths(
+                gcc_vs_gcc_refrstdir,
+                "Restart",
+                all_months_ref
+            )[0]
+            dev_datafiles = get_filepaths(
                 gcc_vs_gcc_devrstdir,
                 "Restart",
                 all_months_dev
             )[0]
 
-            # Pick output folder
-            if config["options"]["comparisons"]["gchp_vs_gcc"]["run"]:
-                tablesdir = gchp_vs_gcc_tablesdir
-            else:
-                tablesdir = gcc_vs_gcc_tablesdir
+            # Ref
+            make_benchmark_mass_conservation_table(
+                ref_datafiles,
+                config["data"]["ref"]["gcc"]["version"],
+                dst=gcc_vs_gcc_tablesdir,
+                overwrite=True,
+                spcdb_dir=spcdb_dir,
+            )
 
-            # Create table
-            bmk.make_benchmark_mass_conservation_table(
-                datafiles,
-                config["data"]["dev"]["gcc"]["dir"],
-                dst=tablesdir,
+            # Dev
+            make_benchmark_mass_conservation_table(
+                dev_datafiles,
+                config["data"]["dev"]["gcc"]["version"],
+                dst=gcc_vs_gcc_tablesdir,
```

Comment:
The mass conservation tables especially would benefit from printing Ref and Dev to the same file since there is only one column for each.

Reply:
Agreed

Reply:
@msulprizio: This is now a feature request in #308